spec/pushmi_pullyu/aip/solr_fetcher_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+RSpec.describe PushmiPullyu::AIP::SolrFetcher do
```

Comment:
Missing solr_fetcher unit tests, added this in. (of course its being tests via downloader class, but doesn't hurt to unit test this itself)